package.json

```diff
@@ -8,7 +8,8 @@
     "todomvc-common": "^1.0.2"
   },
   "devDependencies": {
-    "@percy/puppeteer": "^1.1.0",
+    "@percy/cli": "^1.0.0-beta.36",
+    "@percy/puppeteer": "^2.0.0-beta.3",
```

Comment:
I think it's fine to merge this now, with the beta. Dependabot should come through and get us to the final release once it's out. Thoughts?